### ISSUE TITLE
Full pdf version

### ISSUE THREAD

**Issue by lucgiffon** [ethanweed/pythonbook#34]
It would be very nice to have a full pdf version of this thing, with a proper, complete summary.... for offline reading or even printing. I always use the Danielle Navarro's R pdf version, with the complete summary, to find the precise piece that I need, and then go back to the python version.

Also, a real, big, thank you for this work

**Comment by ethanweed** [comment]
Yes, I would like to make a pdf available. I currently have two problems in this regard - since I'm still working on it (and this kind of thing is probably never truly done), I'd like to have a way for people to generate a pdf of the most recent version. Jupyterbook gives you a nice button to make a pdf, but it only gives you the most recent version. Maybe that can be changed. Alternatively, I could probably add a pandoc command to my build-and-commit script to make a pdf and add that to the repo. I will look into this when I get a chance. Thanks for the suggestion!
